### ISSUE TITLE
Add "drone repo update" to update pull request webhook

### ISSUE THREAD

**Issue by Lelenaic** [drone/drone-cli#133]
Hello,
I would like to update like 50-60 drone repos to disable the pull request webhook (which is possible via the web interface). But I cannot do this with the Drone CLI.

Can we add something like :
`drone repo update my-repo --pull-request-webhook=false`

Thanks.

**Comment by bradrydzewski** [comment]
Yes, I would accept a pull request that adds `--ignore-forks` and `--ignore-pull-requests`
